app/assets/javascripts/ama_validators/country_province_selector.js.coffee.erb

```diff
@@ -0,0 +1,33 @@
+class window.CountryProvinceSelector
+  constructor: (@country_select, @province_select) ->
```

Comment:
can we change this to take a hash instead (so the signature doesn't change if we add more params)?

    constructor: (args) ->
        @blah = args.blah || someDefaultBlah
        @foo= args.foo || someDefaultFoo